Source/Parallelization/WarpXComm.cpp

```diff
@@ -982,23 +1071,26 @@ WarpX::FillBoundaryAux (int lev, IntVect ng)
 }
 
 void
-WarpX::SyncCurrent (
-    const amrex::Vector<std::array<std::unique_ptr<amrex::MultiFab>,3>>& J_fp,
-    const amrex::Vector<std::array<std::unique_ptr<amrex::MultiFab>,3>>& J_cp,
-    const amrex::Vector<std::array<std::unique_ptr<amrex::MultiFab>,3>>& J_buffer)
+WarpX::SyncCurrent (const std::string& current_fp_string)
 {
+    using ablastr::fields::Direction;
+
     WARPX_PROFILE("WarpX::SyncCurrent()");
 
+    ablastr::fields::MultiLevelVectorField const& J_fp = m_fields.get_mr_levels_alldirs(current_fp_string, finest_level);
+
     // If warpx.do_current_centering = 1, center currents from nodal grid to staggered grid
     if (do_current_centering)
     {
+        ablastr::fields::MultiLevelVectorField const& J_fp_nodal = m_fields.get_mr_levels_alldirs(FieldType::current_fp_nodal, finest_level+1);
```

Comment:
Passing `finest_level+1` instead of `finest_level` here might have been a mistake. This came up in #5356 and I will suggest a fix there.